telethon/client/chats.py

```diff
@@ -1145,6 +1145,23 @@ async def kick_participant(
         else:
             raise ValueError('You must pass either a channel or a chat')
 
+    async def get_permissions(
+            self: 'TelegramClient',
+            channel: 'hints.EntityLike',
```

Comment:
```suggestion
            entity: 'hints.EntityLike',
```

Reply:
We have to rename from `channel` to `entity`, because either chat or channel

Reply:
It's also consistent with the rest of methods, so I'm fine with this.